pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "acih-backend"
+version = "0.1.0"
```

Comment:
Can we put "0.0.0" here? Since we didn't have any releases yet

Reply:
Alright, according to [this FAQ](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase) it would better to keep it "0.1.0" at the initial development phase: 

> **How should I deal with revisions in the 0.y.z initial development phase?**
>
> The simplest thing to do is start your initial development release at 0.1.0 and then increment the minor version for each subsequent release.

So we don't need the change proposed by me.